content/chat/rooms/occupancy.textile

```diff
@@ -6,23 +6,27 @@ languages:
   - javascript
 ---
 
-Occupancy enables you to view the number of users in a room. This feature can be used to display room counts or highlight popular, or trending chat rooms.
+Occupancy enables you to view the number of users currently online in a room. This feature can be used to display user counts to highlight popular, or trending chat rooms.
 
 <aside data-type='note'>
 <p>Occupancy needs to be "enabled":/chat/rooms#create when creating or retrieving a room.</p>
 </aside>
 
 h2(#subscribe). Subscribe to room occupancy
 
-Subscribe to a room's occupancy by registering a listener. Occupancy events are emitted whenever the membership of a room changes. Use the "@occupancy.subscribe()@":https://sdk.ably.com/builds/ably-labs/ably-chat-js/main/typedoc/interfaces/Occupancy.html#subscribe method in a room to receive updates:
+Subscribe to a room's occupancy by registering a listener. Occupancy events are emitted whenever the number of online users within a room changes. Use the "@occupancy.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/Occupancy.html#subscribe method in a room to receive updates:
 
 ```[javascript]
 const {unsubscribe} = room.occupancy.subscribe((event) => {
   console.log(event);
 });
 ```
 
-h3(#payload). Occupancy payload
+<aside data-type='note'>
+<p>Occupancy events are debounced for no more than 15 seconds. If a mode change has ocurred then the change is emitted immediately, for example at least one connection now existing where none did previously.</p>
```

Comment:
Is this documented elsewhere as well? Since our occupancy is a wrapper, should we link to the occupancy feature in channels which will give more details about all things?

Reply:
It is covered in the existing occupancy docs, but the issue is that there's a lot of additional things that we've removed. I'm not sure if it would be more confusing to include the link because of this.